spec/models/comment_spec.rb

```diff
@@ -37,4 +42,73 @@
     comment = build(:comment, markeddown_comment: "a" * 16_777_216)
     expect(comment).to_not be_valid
   end
+
+  it "sends reply notification" do
+    # The user receiving the notification.
+    u = create(:user)
+    u.settings["email_notifications"] = true
+    u.settings["email_replies"] = true
+
+    # The user sending the notification.
+    u2 = create(:user)
+    # Story under which the comments are posted.
```

Comment:
You don't need to create `s` explicitly, creating a comment without specifying a story will implicitly create one.

Reply:
Wouldn't the parent and child comments need to share a story? Or does that happen automatically when I pass the parent comment?